app/controllers/admin/articles_controller.rb

```diff
@@ -37,7 +37,7 @@ def destroy
     private
 
     def article_params
-      params.require(:article).permit(:title, :description, :content)
+      params.require(:article).permit!
```

Comment:
__[Brakeman]__

Parameters should be whitelisted for mass assignment


<!-- Comment by [SideCI](https://sideci.com) -->